### PR TITLE
docs(success stories): Add Adobe conference talk to videos area

### PR DESCRIPTION
**Pull request by dnilasor**
This is a recent company testimonial (late last year) from the Adobe Experience Platform team. I'd like to add it to success stories. It's replacing a Netflix video, since there is already a Netflix case study on the page. @brian-armory redone and closing the other, thx

### CODE REVIEW

File: success-stories/index.html [hunk 1]
```diff
@@ -91,9 +91,9 @@ <h6>Conference talks, interviews, and video testimonials from the Spinnaker Comm
     </div></div>
 
     <div class="video-item">
-    <p>Andy Glover: How Netflix Runs Spinnaker @ Scale</p>
+    <p>Adobe Deploys Critical Apps With Spinnaker </p>
```
Comment (dorbin) [inline success-stories/index.html:94]: @dnilasor , replacing Andy's video rather than adding?

Comment (dnilasor) [reply success-stories/index.html:94]: pre-site-refresh, yes. I thought the SIG agreed we don't want too many videos on there, and I am not able to design/implement a better videos area on this page for now. We need multiples of 3 for what I hacked together to work. Netflix has a prominent use case on the page already and I think for this page, adding companies that aren't clearly associated with Spinnaker success already is a priority. 

@dorbin open to alternatives or to removing a different video.

Comment (dorbin) [reply success-stories/index.html:94]: 👍 SGTM!

**Review by 317brian — APPROVED** [review]
Since @dorbin already chimed in, I'm going to go ahead and merge. Built it locally, and it looks fine.

**Pull request merged**